<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogFile</class>
 <widget class="QWidget" name="LogFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="MyPlainTextEdit" name="plainTextEdit">
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="refresh">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="qressource.qrc">
         <normaloff>:/Toolbar/Refresh.png</normaloff>:/Toolbar/Refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="stop">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="qressource.qrc">
         <normaloff>:/Toolbar/player_pause.png</normaloff>:/Toolbar/player_pause.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clear">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="qressource.qrc">
         <normaloff>:/Toolbar/edit-clear.png</normaloff>:/Toolbar/edit-clear.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboLevels">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="findWidgetContainer" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>780</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>myplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qressource.qrc"/>
 </resources>
 <connections/>
</ui>
